platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/pipeline/VirtualPipeline.java

```diff
@@ -562,8 +561,10 @@ private void hashFlushMerge() {
                 logger.debug(VIRTUAL_MERKLE_STATS.getMarker(), "Merge {}", copy.getFastCopyVersion());
                 merge(next);
                 copies.remove(next);
-                statistics.setPipelineSize(copies.getSize());
```

Comment:
This code is moved two lines below, outside of the block, so it's now executed both for "flush" and "merge" cases